schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
-
+/*Milestone 01 */
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100),
+    date_of_birth date,
+    escape_attempts INT,
+    n eutered BOOLEAN,
```

Comment:
- [ ] You have a typo here, kindly make sure to remove the space so the column name is neutered